src/elements/content-preview/ContentPreview.js

```diff
@@ -1110,21 +1119,28 @@ class ContentPreview extends PureComponent<Props, State> {
         if (!currentFileId) {
             return null;
         }
+
+        const currentVersionId = getProp(file, 'file_version.id');
+        const selectedVersionId = getProp(selectedVersion, 'id', currentVersionId);
+
+        const onHeaderClose = currentVersionId === selectedVersionId ? onClose : this.updateVersionToCurrent;
```

Comment:
You may want to add a check to see if `this.updateVersionToCurrent` is defined.

Reply:
I think this actually follows expected behavior which is that if its undefined the header won't render a close button.